tests/integration/kubernetes/setup.sh

```diff
@@ -102,10 +103,25 @@ add_cbl_mariner_kernel_initrd_annotations() {
 	fi
 }
 
+add_runtime_handler_annotations() {
```

Comment:
Did you try running the tests without this? With Chengyu's latest changes we found it was not required anymore to get image pull

Reply:
I did, and it worked.  The reason it worked is kubernets 1.29+, which is correctly passing down the runtime handler information as part of the image pull request.

However, with 1.27 and 1.28 still being supported, I'm not so happy to just drop that from our tests.

If you prefer, I can drop it, and then we find a place to properly document the behaviour, which  ... thinking here ... may be the most appropriate approach.  So, I will drop it as soon as you're finished with the review.

Reply:
Ah ok - I didn't realist that was the distinction - I bumped our peer pods cluster from 1.26 to 1.30 last week, so maybe the timing just synced with Chengyu's fix. I'm fine to leave it in based on this, but maybe mention in a comment it is for 1.27 & 1.28 support and then we can maybe remember to clean it up in October when they EoL?